### ISSUE TITLE
How to transform the trained pointpillars model into onnx model needed by Apollo6.0

### ISSUE THREAD

**Issue by qiuhui1991** [ApolloAuto/apollo#13770]
only need transform rpn.pt to rpn.onnx ?

**Comment by jeroldchen** [comment]
@qiuhui1991 No, onnx model is exported by `torch.onnx.export` . To learn more details about exporting onnx model, you can check out this tutorial: https://pytorch.org/tutorials/advanced/super_resolution_with_onnxruntime.html

**Comment by qiuhui1991** [comment]
thank you for your reply




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ApolloAuto/apollo"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年5月13日(星期四) 下午2:48
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ApolloAuto/apollo] How to transform the trained pointpillars model into onnx model needed by Apollo6.0 (#13770)





 
@qiuhui1991 No, onnx model is exported by torch.onnx.export . To learn more details about exporting onnx model, you can check out this tutorial: https://pytorch.org/tutorials/advanced/super_resolution_with_onnxruntime.html
